client/my-sites/themes/controller.jsx

```diff
@@ -31,6 +32,7 @@ export function getProps( context ) {
 }
 
 export function loggedOut( context, next ) {
+	performanceMark( context, 'loggedOut' );
```

Comment:
Should we consider adding some more specificity to such names? There could potentially be more `loggedOut` marks and it could cause confusion, so it would be useful to know that it belongs to the `themes` section.

Reply:
I don't think we need to for two reasons:

1. In the ultimate log to logstash, the section and route is recorded. So we can look at records for _just_ /themes.
2. We don't really want dynamic names for the "marks." This is because the logstash index can only support so many keys, which means we don't want to inflate that number with things like "themes-loggedOut".

And ultimately, my goal is to look at individual records rather than aggregates. So as long as the individual record includes all the info we need, I think it's ok :)

Reply:
i think you have a fair point about this specific name which I missed at first though! :) I'll double check a couple of these names and make sure they're clearer.

Reply:
updated!